RELEASE-NOTES.txt

```diff
@@ -1,6 +1,12 @@
 Unreleased
 ==========
 
+1.5.5
+=====
+* Improved the pull and push functionality to allow selective sync
+* Misc UI fixes #1538, #1533, #1534, #1494
+* DX fixes #1430, #1489
```

Comment:
What is DX? Can we make it less cryptic?

Reply:
Developer experience - these PRs around improving something for development. As option, maybe we shoudl not include it to release notes, as it's not for customers, WDYT?